api/src/main/java/org/datacleaner/api/AnalyzerResultFutureImpl.java

```diff
@@ -165,9 +167,17 @@ public boolean isReady() {
     public R get() {
         if (_countDownLatch != null) {
             try {
-                _countDownLatch.await();
+                boolean finished = false;
+                int iteration = 0;
+                while (!finished) {
+                    iteration++;
+                    finished = _countDownLatch.await(2, TimeUnit.MINUTES);
+                    if (!finished) {
+                        logger.info("Awaited completion for " + (iteration * 2) + " minutes...");
+                    }
+                }
```

Comment:
Yes because this class is in the api module where the utils are not available. Kinda lame but I found only this one case :-/